nidata/multimodal/hcp/__init__.py

```diff
@@ -214,10 +248,10 @@ def get_files(self, data_type, volume_type, subj_id):
         else:
             raise NotImplementedError("Cannot (yet!) fetch '%s' files" % (
                 volume_type))
-        return files
+        return files"""
```

Comment:
Never comment out code in git--it's versioned, so you have old copies of the code in past commits. Please delete the commented out code.